package.json

```diff
@@ -16,14 +16,17 @@
     "unsafe-perm": true
   },
   "scripts": {
-    "dev": "webpack -d --progress --profile --colors --bail --",
-    "serve": "webpack-dev-server -d --inline --https --content-base troposphere/assets",
-    "build": "webpack --progress --profile --colors --bail --",
-    "watch": "webpack --progress --profile --colors --bail --watch -d --",
+    "clean": "rimraf troposphere/assets/bundles",
+    "serve": "webpack-dev-server --hot --env.webpack=development",
+    "build:dev": "npm run clean && webpack --env.webpack=development",
+    "build:prod": "npm run clean && webpack --env.webpack=production -p",
     "jenkins": "webpack --profile --colors --",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "stats": "webpack --profile --json > stats.json",
-    "lint": "eslint -c .eslintrc --ext .jsx,.js --no-eslintrc troposphere/static/js",
+    "stats:dev": "webpack --json --env.webpack=development > stats.json",
+    "stats:prod": "webpack --json --env.webpack=production -p > stats.json",
+    "lint": "eslint troposphere/static/js",
```

Comment:
This region has changed and is changing in #654 & #659. 

We want `"lint"` to be:
```
"lint": "eslint -c .eslintrc --ext .jsx,.js --no-eslintrc troposphere/static/js",
```